zrml/prediction-markets/src/lib.rs

```diff
@@ -131,6 +137,7 @@ mod pallet {
             };
             if market_status == MarketStatus::Proposed {
                 slash_market_creator(T::AdvisoryBond::get());
+                MarketIdsForEdit::<T>::remove(market_id);
```

Comment:
you mean to create a new case for Proposed market in benchmark `admin_destroy_market()` ?

Reply:
if we do that how to use it with .max() for weight above?

Reply:
Oh this was just an open thought. Because I don't think, that remove reads the value of the storage, it's okay to ignore it in the benchmarks. But if anyone of you know why, and that this remove actually should be regarded in the benchmarks, feel free to correct me. 

So the question is:

MarketIdsForEdit = [0, 1, 2]

MarketIdsForEdit::remove(3)

Does this different in case of the weight than this MarketIdsForEdit::remove(1)? 

Especially when behind 1 is a big list of length MaxEditReasonLen.

You could create a benchmark path for Proposed markets, yeah, but I don't think that it exceeds the weight of  reported or disputed markets. And because of each market status other than reported and disputed it takes the Default weight, so the maximum weight of reported and disputed I could live with as it is now. The admin_destroy_market is only callable by DestroyOrigin anyways, so it's even more  irrelevant. So, forget my comment. I just wanted to figure out, what actually happens for remove in the cases explained above.